docs/how-to-guides/new-user-guides/authentication-permissions-and-global-configuration/create-pod-security-policies.md

```diff
@@ -2,9 +2,13 @@
 title: Pod Security Policies
```

Comment:
```suggestion
title: Pod Security Standards (PSS) & Pod Security Admissions (PSA)
```

See my comment further down about the Rancher GUI.

Also:  It's just hard to talk about either resource without mentioning the other.

It's tempting to pull PSACT content into its own page, since how-to create and edit PSACTs feels like its own topic. That leaves the PSS page with the intro, a section on migration, and a section on Rancher namespace exemptions. Thoughts?